src/Object/Product/Product.php

```diff
@@ -980,4 +977,33 @@ public function isValid()
             return false;
         }
     }
+
+    /**
+     * Makes the alternative images collection unique and removes
+     * the main product image from alt images if present
+     */
+    private function altImagesToUnique()
+    {
+        if ($this->alternateImageUrls->count() == 0) {
```

Comment:
Strict types may be better
```suggestion
        if ($this->alternateImageUrls->count() === 0) {
```